website/www/site/layouts/partials/section-menu/en/sdks.html

```diff
@@ -39,6 +39,7 @@
     <li><a href="/documentation/sdks/python-dependencies/">Python SDK dependencies</a></li>
     <li><a href="/documentation/sdks/python-streaming/">Python streaming pipelines</a></li>
     <li><a href="/documentation/sdks/python-type-safety/">Ensuring Python type safety</a></li>
+    <li><a href="/documentation/sdks/python-machine-learning/">Machine Learning</a></li>
```

Comment:
should we also this content at https://beam.apache.org/documentation/resources/learning-resources/#machine-learning ?

Reply:
Yeah, that's a good idea. I'll add that.